src/main/java/uk/gov/hmcts/reform/hmc/config/MessageSenderToQueueConfiguration.java

```diff
@@ -32,9 +30,7 @@ public void sendMessageToQueue(String message, Long hearingId, String messageTyp
             log.debug("Connected to queue {}", applicationParams.getInternalOutboundQueueName());
             ServiceBusMessage serviceBusMessage = new ServiceBusMessage(message);
             serviceBusMessage.getApplicationProperties().put(MESSAGE_TYPE, messageType);
-            if (AMEND_HEARING.equals(messageType) || DELETE_HEARING.equals(messageType)) {
-                serviceBusMessage.getApplicationProperties().put(HEARING_ID, hearingId);
-            }
+            serviceBusMessage.getApplicationProperties().put(HEARING_ID, hearingId);
```

Comment:
The sync response that Outbound Adapter gets needs to pass the hearing ID back, so now adding this as part of all flows.